### PR TITLE
JDBC Connector version consolidation

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Only publishing current version - older versions being consolidated

### CODE REVIEW

File: product_docs/docs/jdbc_connector/42.2.24.1/02_requirements_overview.mdx [hunk 1]
```diff
@@ -5,36 +5,15 @@ title: "Requirements Overview"
...
--   Windows Server 2012 R2
--   Windows 10
--   Windows 8.1
-
-32-bit Windows:
-
--   Windows 10
--   Windows 8.1
+This table lists the latest JDBC Connector versions and their supported corresponding EDB Postgres Advanced Server (EPAS) versions.  Pgpool-II is supported on the same platforms as EDB Postgres Advanced Server. See [Product Compatibility](https://www.enterprisedb.com/platform-compatibility#epas) for details.
+
```
Comment (dwicinas) [inline product_docs/docs/jdbc_connector/42.2.24.1/02_requirements_overview.mdx:9]: @drothery-edb do you want "Pgpool-II" in that text? I assume no.

**Comment by josh-heyer** [comment]
Added legacy redirects.

**Review by ericmack — APPROVED** [review]

**Review by ericmack — APPROVED** [review]
Viewed the test deploy and all looks good.

**Comment by drothery-edb** [comment]
@josh-heyer , it is ok to merge now, correct? Looks like we are all set with the redirects.

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**